### ISSUE TITLE
UWP support?

### ISSUE THREAD

**Issue by JimSEOW** [TadasBaltrusaitis/OpenFace#209]
It seems that the key libraries: [openCV UWP Nuget ](https://github.com/cvwin/cvwin)and Dlib [can be compiled for UWP] are possible in UWP. 

I hope there will be eventually a UWP version using C# (Managed c++ support)

**Comment by TadasBaltrusaitis** [comment]
I am not as familiar with UWP, but there is a work in progress version of OpenFace that calls it from C# using managed C++ wrappers - https://github.com/TadasBaltrusaitis/OpenFace/tree/Windows-GUI

**Comment by JimSEOW** [comment]
@TadasBaltrusaitis 
Yes, I have looked into it.  I post here perhaps others may also show interest.
This is what I have gathered:
Nuget: https://www.nuget.org/packages/OpenCV.Win.Core/ provides

Once Nuget Install e.g. in an empty VS2015 UWP C++ project.
In the packages folder, we get win10-arm native dll and static lib
- opencv_core310.dll
- opencv_core310d.dll
- opencv_core310.lib
- opencv_core310d.lib

I still learning how OpenFace with the C++ wrappers are organized. 

DLib compile without problem in VS2015 Arm 

WIth both, I think we have all elements for making OpenFace ready for Q4 2017: dawn of Windows on ARM
